tests/utils.py

```diff
@@ -97,7 +97,7 @@ def formatError(self, test, err):
 
 		_, module, call = test.address()
 
-		output = ['python2', 'run_tests.py', u'%s:%s' % (module, call)]
+		output = ['python2', 'run_tests.py', u'{0}:{1}'.format(module, call)]
```

Comment:
Yeah, `"{module}:{call}".format(module=module, call=call)` is definitly too much. And also, that's a good way to convince me to keep `"%s:%s" % (module, call)`. :D

But `"{}:{}".format(module, call)` is fine I guess.